onnxruntime/core/mlas/lib/pooling.cpp

```diff
@@ -1197,7 +1197,7 @@ Return Value:
     bool AllKernelsAreSmall = true;
 
     if (Dimensions > 3) {
-        throw std::runtime_error("bad dimensions");
+        ORT_THROW("bad dimensions");
     }
```

Comment:
We may want a separate macro and cmake option here instead of using ORT values. Not sure if we should be ensuring MLAS is completely decoupled from ORT so it's a standalone product. @tracysh for guidance.